doc/rados/configuration/bluestore-config-ref.rst

```diff
@@ -78,15 +80,15 @@ of the device), then the :ref:`ceph-volume-lvm` call for an lv named
 
 **block and block.db**
 ^^^^^^^^^^^^^^^^^^^^^^
-If there is a mix of fast and slow devices (spinning and solid state),
+If you have a mix of fast and slow devices (SSD / NVMe and rotational),
 it is recommended to place ``block.db`` on the faster device while ``block``
-(data) lives on the slower (spinning drive). Sizing for ``block.db`` should be
-as large as possible to avoid performance penalties otherwise. The
-``ceph-volume`` tool is currently not able to create these automatically, so
-the volume groups and logical volumes need to be created manually.
+(data) lives on the slower (spinning drive).
 
-For the below example, lets assume 4 spinning drives (sda, sdb, sdc, and sdd)
-and 1 solid state drive (sdx). First create the volume groups::
+You must create these volume groups and logical volumes manually as 
```

Comment:
Isn't "these" word redundant in this sentence?

Reply:
In my opinion, "these" is not redundant. In this context, it emphasizes to the reader the the volume groups under discussion are the particular ones in this example.

Reply:
My intent exactly, Zac.  I struggled with this wording for a while, I for sure wanted to make it more clear.  This is another of those things that I see people on the list tripping over.